hack/lib/clone_jobs.py

```diff
@@ -0,0 +1,29 @@
+import json, sys, yaml, copy;
+
+def duplicate_job(job, from_branch, to_branch, to_version):
+  if 'branches' not in job:
+    return []
+  for name in job['branches']:
+    if name == from_branch:
+      n = copy.deepcopy(job)
+      n['branches'] = [to_branch]
+      n['name'] += "-" + to_version
```

Comment:
Let's `pull-ci-$org-$repo-$branch-$name`

Reply:
I agree, but I don't have the info here to reconstruct that (I'd have to set up arbitrary rules for parsing).  When we have the generator we can make this better, but for now I'm going to replace "master" with the branch, or just append the branch.

Reply:
Example:

```
pull-ci-kubernetes-cluster-capacity-unit-release-3.11
```

If `master` was in that, it would get replaced with `release-3.11` instead of appended.